pkg/database/database.go

```
@@ -43,7 +43,7 @@ type ConnectOptions struct {
 	Database string
 
 	User     string
-	Password string
+	Password string //nolint:gosec // We need a password to connect to the database.
```

golangci-lint (2.5.0)

[error] directive `//nolint:gosec // We need a password to connect to the database.` is unused for linter "gosec"

(nolintlint)